### PR TITLE
added hailer24.json

### PR DESCRIPTION
**Pull request by hailer24**
## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1034"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]
Looks good to me! Thanks for contributing!

**Review by CodingSpecies — APPROVED** [review]

File: public/data/hailer24.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      "url": "https://www.linkedin.com/in/anmol-bhutani-b132931a5/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Take a look at my portfolio",
+      "url": "https://mole24.netlify.app/",
+      "icon": "Link"
+    },
+    {
+      "name": "check my Codeforces account",
```
Comment (Panquesito7) [inline public/data/hailer24.json:23]: I think it'd look better if you capitalize `c` here 😅 what do you think? Thanks.
```suggestion
      "name": "Check my Codeforces account",
```

File: public/data/hailer24.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      "title": "Started Web development",
+      "date": "June 2021",
+      "icon": "Globe",
+      "color": "blue"
+    },
+    {
+      "title": "Joined NIT Kurukshetra",
+      "date": "July 2019",
+      "icon": "Graduation Hat",
```
Comment (krshdev) [inline public/data/hailer24.json:38]:
```suggestion
      "icon": "graduation-hat",
```

File: public/data/hailer24.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      "name": "check my Codeforces account",
+      "url": "https://codeforces.com/profile/mole24",
+      "icon": "Codeforces"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Web development",
+      "date": "June 2021",
+      "icon": "Globe",
```
Comment (krshdev) [inline public/data/hailer24.json:32]:
```suggestion
      "icon": "globe",
```

File: public/data/hailer24.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    {
+      "name": "Take a look at my portfolio",
+      "url": "https://mole24.netlify.app/",
+      "icon": "Link"
+    },
+    {
+      "name": "check my Codeforces account",
+      "url": "https://codeforces.com/profile/mole24",
+      "icon": "Codeforces"
```
Comment (krshdev) [inline public/data/hailer24.json:25]:
```suggestion
      "icon": "codeforces"
```

File: public/data/hailer24.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    {
+      "name": "Conenct with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/anmol-bhutani-b132931a5/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Take a look at my portfolio",
+      "url": "https://mole24.netlify.app/",
+      "icon": "Link"
```
Comment (krshdev) [inline public/data/hailer24.json:20]:
```suggestion
      "icon": "link"
```

File: public/data/hailer24.json (hunk 2 above)
Comment (emmalearnscode) [reply public/data/hailer24.json:38]: This is an unnecessary change, both will work in the code as we convert everything to lowercase

Comment (krshdev) [reply public/data/hailer24.json:38]: but the dash @emmalearnscode ?

File: public/data/hailer24.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      "name": "Check my Codeforces account",
+      "url": "https://codeforces.com/profile/mole24",
+      "icon": "codeforces"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Web Development",
+      "date": "June 2021",
+      "icon": "Globe",
```
Comment (Panquesito7) [inline public/data/hailer24.json:32]: Shouldn't we use `globe` here without capitalizing it?

File: public/data/hailer24.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+      "color": "blue"
+    },
+    {
+      "title": "Joined NIT Kurukshetra",
+      "date": "July 2019",
+      "icon": "Graduation-hat",
```
Comment (Panquesito7) [inline public/data/hailer24.json:38]: Same here.

Comment (hailer24) [reply public/data/hailer24.json]: changed to lowercase

File: public/data/hailer24.json (hunk 6 above)
Comment (hailer24) [reply public/data/hailer24.json]: changed to lowercase

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, Thanks for your contribution :)

File: public/data/hailer24.json (hunk 2 above)
Comment (emmalearnscode) [reply public/data/hailer24.json]: Ok, sorry, I missed that @krshdev .

**Pull request merged**